test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] I think there is an issue with this file because it's supposed to be ignored since it's in your `.gitignore` file. The reason could be that you tracked it before ignoring it and to fix this you could try cleaning the git cache :pushpin: :wink: